src/constants.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+from dotenv import dotenv_values
+
+constants = {
+    **dotenv_values(".env.shared"),  # load shared development variables
+    **dotenv_values(".env.secret"),  # load sensitive variables
+    **os.environ,  # override loaded values with environment variables
+}
```

Comment:
Should we add a test to check if an environment variable appears in the constants dict?

Reply:
Sounds reasonable. An other issue that might be worth testing for: The preference of loaded env-vars (`os.environ` is more important than `env.secret` is more important than `env.shared`; in case someone restructures this code later).

Reply:
That sounds reasonable 👍 I'll add it right away